recipe/meta.yaml

```diff
@@ -1,5 +1,8 @@
 {% set name = "mesaLib" %}
-{% set version = "17.3.9" %}
+{% set version = "17.3.9" %}  # [linux]
+# 17.1 uses fstatat which isn't defined on OSX.
+# https://github.com/conda-forge/mesalib-feedstock/pull/18#issuecomment-450589848
```

Comment:
`fstatat` isn't present on macOS until 10.10. We are building for 10.9 currently. So working around this somehow is reasonable.

Homebrew has run into [similar issues with other packages]( https://bugs.archlinux.org/task/49771 ).

We probably should try to get the same version on Linux and macOS before merging anything. Though I'm guessing you are just trying to get something working in the first place. So we can save that concern for later. :)